styles/_base.scss

```diff
@@ -117,48 +117,53 @@ ul, ol {
 
   #contentFrame {
     position: relative;
-    padding-top: $barLg;
     overflow-y: scroll;
     height: calc(100% - #{$bar});
     box-sizing: border-box;
     z-index: 0;
 
-    #pageTabBar {
-      position: fixed;
-      top: $bar;
-      left: 0;
-      z-index: 1;
-      width: 100%;
-      height: $barLg;
-      border-bottom-width: 1px;
-      border-bottom-style: solid;
-
-      .pageTabs {
-        max-width: $pageWidth;
-        margin: 0 auto;
-
-        .userIcon {
-          border-width: 2px;
-          border-style: solid;
-          width: 44px;
-          height: 44px;
-          margin-right: $pad;
-        }
+    .chatOpen &:before {
+      right: 0;
+    }
+
+    #pageContainer {
+      padding-top: $barLg;
+
```

Comment:
Previously the pageTabBar style was nested in contentFrame instead of pageContainer, this fixes the hierarchy, and puts the padding-top on pageContainer which makes layout more consistent.